lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}
+
+# Define the location of your websockets here
+<%= application_name %>.cable = Cable.createConsumer 'ws://localhost:28080'

```

Comment:
Agreed. That's something that can nearly always be inferred from the host protocol as browsers won't allow mixed content.

It shouldn't be too hard to determine if `xss://` or `xs://` has already been included using a regex, like `/^xss?:\/\//.test(url)`. Wouldn't need an option at all at that point.